drop-in/src/main/java/com/adyen/checkout/dropin/ui/viewmodel/DropInSavedStateHandleContainer.kt

```diff
@@ -75,6 +76,7 @@ internal object DropInBundleHandler {
             packageName,
         )
         intent.apply {
+            putExtra(CHECKOUT_SESSION_KEY, checkoutSession)
```

Comment:
Having both `checkoutSession` and `checkoutSession.sessionSetupResponse` is redundant, let's remove the previous

Reply:
Or actually maybe we don't need to pass the whole `CheckoutSession` object to the providers. How about we only pass the `SessionSetupConfiguration` object, and we store it inside `SessionDetails` (just like the rest of the fields. Passing the whole session just to take one field seems overkill

Reply:
I agree with you, I am just concerned that we might need something else inside the `ComponentParams` not just  `SessionSetupConfiguration`

Reply:
Yeah I agree but it seems like the API only wants to use this object to configure components, I guess we'll refactor it if that changes 🤷